tests/e2e/global.go

```diff
@@ -16,8 +16,30 @@ package e2e
 
 import (
 	"errors"
+	"os"
+	"strings"
 )
 
+func init() {
+	// Set helm chart values for Koperator to be able to use custom image
+	koperatorImagePath := os.Getenv("IMG_E2E")
+	if koperatorImagePath != "" {
+		koperatorImagePathSplit := strings.Split(koperatorImagePath, ":")
+
+		koperatorImageRepository := koperatorImagePathSplit[0]
+		koperatorImageTag := "latest"
+
+		if len(koperatorImagePathSplit) == 2 {
+			koperatorImageTag = koperatorImagePathSplit[1]
+		}
+
+		koperatorLocalHelmDescriptor.SetValues = map[string]string{
+			"operator.image.repository": koperatorImageRepository,
+			"operator.image.tag":        koperatorImageTag,
+		}
+	}
+}
+
```

Comment:
**Optional**: I **MAY** have opted to use an inline local lambda function on the Helm descriptor variable setting to do this on the `koperatorLocalHelmDescriptor`. IMO it makes the logic local to there and still fulfills the purpose because the global var init runs at package loading.

Reply:
Good idea thank you!